front/src/modules/users/components/Avatar.tsx

```diff
@@ -95,7 +110,8 @@ export function Avatar({
       type={type}
       colorId={colorId}
     >
-      {noAvatarUrl && placeholder[0]?.toLocaleUpperCase()}
+      {(noAvatarUrl || isInvalidAvatarUrl) &&
+        placeholder[0]?.toLocaleUpperCase()}
```

Comment:
Hey @charlesBochet,
How about we let the avatar determine whether to display a letter or an image? I believe this is a lot simpler.

Reply:
Yes, that makes sense